### PR TITLE
Update-controller-plugin.md

### PR DESCRIPTION
**Pull request by chaytanyasinha**
Description
I am improving the doc in controller-plugin.md by adding a small how to create your own robot window.

This pull-request fixes issue #1055

### CODE REVIEW

File: docs/guide/controller-plugin.md [hunk 1]
```diff
@@ -68,12 +68,21 @@ Robot windows are implemented in HTML and provide the following features:
...
 3. They are web-ready and could be used to display robot windows on web pages.
 
-A simple example of an HTML robot window is provided in the `robots/thymio/thymio2.wbt` sample simulation and demonstrates:
+A simple example of an HTML robot window is provided in the `custom_robot_window` sample simulation and demonstrates:
 
 - How to display sensor information in the robot window.
 - How to send user interface events (like mouse clicks) from the robot window to the controller program.
 - How to change the title of the robot window from the controller program.
 
+How to create your own robot-window:
```
Comment (DavidMansolino) [inline docs/guide/controller-plugin.md:77]:
```suggestion
#### How to Create Your Own Robot-window:
```

**Comment by chaytanyasinha** [comment]
@DavidMansolino Can you please tell me any resources which I can refer for the changes

**Comment by DavidMansolino** [comment]
Unfortunately such references does not exist yet (this is the reason of the issue), but here is an example of simple custom robot-window: https://github.com/cyberbotics/webots/tree/master/projects/samples/howto/plugins/robot_windows/custom_html_robot_window

**Comment by chaytanyasinha** [comment]
@DavidMansolino Okay.I will create this as a draft proposal so that you can make the required changes after I will make the doc.

**Pull request closed without merging**